server/pulp/server/db/model/dispatch.py

```diff
@@ -221,6 +222,14 @@ def as_dict(self):
             'total_run_count': self.total_run_count,
         }
 
+        for key, value in dict_repr.iteritems():
```

Comment:
You're not really saving anything with `iteritems` here; might as well stick with `items` since `iteritems` is deprecated in py3.